policyengine_us/reforms/tax_foundation/growth_and_opportunity/remove_standard_deduction_head_of_household.py

```diff
@@ -0,0 +1,134 @@
+from policyengine_us.model_api import *
+
+
+def create_remove_standard_deduction_head_of_household() -> Reform:
+    class basic_standard_deduction(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Basic Standard Deduction (Eliminating HoH)"
+        unit = USD
+        documentation = "The Basic standard deduction under the tax foundation growth and opportunity plan"
+        definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        std = parameters(period).gov.irs.deductions.standard
+        filing_status = tax_unit("filing_status", period)
+        statuses = filing_status.possible_values
+        separate_filer_itemizes = tax_unit("separate_filer_itemizes", period)
+        claimed_as_dependent_elsewhere = tax_unit(
+            "tax_unit_dependent_elsewhere", period
+        )
+        if (
+            parameters(
+                period
+            ).gov.contrib.tax_foundation.growth_and_opportunity.remove_head_of_household
+            == True
+        ):
+            standard_deduction = select(
+                [
+                    filing_status == statuses.SINGLE,
+                    filing_status == statuses.SEPARATE,
+                    filing_status == statuses.JOINT,
+                    filing_status == statuses.WIDOW,
+                    filing_status == statuses.HEAD_OF_HOUSEHOLD,
+                ],
+                [
+                    std.amount[filing_status],
+                    std.amount[filing_status],
+                    std.amount[filing_status],
+                    std.amount[filing_status],
+                    13850,  # this should be single amount, use fix amount as temp
```

Comment:
@MaxGhenis how can we extract the single value from a breakdown here? Any suggestions for a clean solution without creating 5 separate files?

Reply:
@MaxGhenis friendly reminder